libs/ui-components/src/components/common/LabelsView.tsx

```diff
@@ -1,14 +1,32 @@
 import * as React from 'react';
-
 import { Label, LabelGroup } from '@patternfly/react-core';
 
-const LabelsView = ({ prefix, labels }: { prefix: string; labels: Record<string, string | undefined> | undefined }) => {
+import { useTranslation } from '../../hooks/useTranslation';
+
+import './LabelsView.css';
+
+interface LabelsViewProps {
+  prefix: string;
+  labels: Record<string, string | undefined> | undefined;
+  variant?: 'default' | 'collapsed';
+}
+
+const LabelsView = ({ variant, prefix, labels }: LabelsViewProps) => {
+  const { t } = useTranslation();
   const labelItems = Object.entries(labels || {});
   if (labelItems.length === 0) {
     return '-';
   }
+
+  const isLimited = variant === 'collapsed';
+
   return (
-    <LabelGroup numLabels={5}>
+    <LabelGroup
+      numLabels={isLimited ? 2 : 5}
+      className={isLimited ? 'fctl-labels-view__collapsed' : undefined}
+      expandedText={t('Show less')}
+      collapsedText={t('${remaining} more')}
```

Comment:
`remaining` needs to be passed as property `count`

Reply:
Actually it's not needed because the `LabelGroup` uses this text as a template and adds the number itself.
However, this could cause confusion to the translators, so I will change it as if it was a normal label so it works as expected.